templates/pgbouncer_info.conf.j2

```diff
@@ -0,0 +1,16 @@
+<LoadPlugin python>
+  Globals true
+</LoadPlugin>
+
+<Plugin python>
+  ModulePath "/usr/share/collectd/python"
+  Import "pgbouncer_info"
+
+  <Module pgbouncer_info>
+    Host "localhost"
+    Port 6432
+    Database "pgbouncer"
+    User "{{ pgbouncer_auth_users[0]['user'] }}"
```

Comment:
This will fail if the auth_user does not have access to pgbouncer. The user needs to be present in pgbouncer.ini as an admin or a stats user.